src/config/dynamic-theme-fixes.config

```diff
@@ -4129,7 +4129,7 @@ INVERT
 ._24NNT
 
 CSS
-._2OVuy{
+.LhRk3{
```

Comment:
Is it possible to change this selector that is less obscure and don't rely on this random "string"?

Reply:
Will try to look for a way, that said webdev and css is black magic for me so if i fail i won't be able to tell for certain.